src/Symfony/Component/Serializer/Normalizer/DenormalizerInterface.php

```diff
@@ -27,6 +27,13 @@
      * @param array  $context options available to the denormalizer
      *
      * @return object
+     *
+     * @throws \Symfony\Component\Serializer\Exception\BadMethodCallException
+     * @throws \Symfony\Component\Serializer\Exception\InvalidArgumentException
+     * @throws \Symfony\Component\Serializer\Exception\UnexpectedValueException
+     * @throws \Symfony\Component\Serializer\Exception\ExtraAttributesException
+     * @throws \Symfony\Component\Serializer\Exception\LogicException
+     * @throws \Symfony\Component\Serializer\Exception\RuntimeException
```

Comment:
Also, we don't document exceptions if we don't also add when this can be thrown.